### DISCUSSION TITLE
Set custom 404 condition per folder?

### DISCUSSION THREAD

**Discussion by aaronrobb** [statamic/cms#6454] [Q&A]
On our site, if there is a missing image and you go to view that missing image it loads the standard 404 page.
This is because the image is loading as a special url, not as a file:

https://www.mysite.com/img/http/aHR0cHM6Ly93d3cudmFuY291dmVyY29udmVudGlvbmNlbnRyZS5jb20vaW1hZ2VzL2xvZ28tZGZkMDdhNzM5Y2NhNmQ2NDVkY2Y0NDg5OWIyNTQ5NWEuanBn?fm=webp&s=e9f5b9c14455c536af4be0228ddc3bb1

Is there a way to specify that if anything in the /img/ folder gets a 404 error, it would just return an empty 404 response and not load the 404 template?

**Comment by jasonvarga** [comment]
I don't think so but you could probably configure the server to use a custom 404 page for a specific directory.

https://nginx.org/en/docs/http/ngx_http_core_module.html#error_page

**Reply by aaronrobb** [reply]
Ok thanks @jasonvarga 
I'm wondering since these urls are set up with Glide if Glide itself has configuration for how to handle missing images?

**Reply by jasonvarga** [reply]
Glide throws a `NotFoundHttpException` which gets caught by the exception handler and then the standard 404 page is rendered. I guess we could throw a more specific `GlideImageNotFoundHttpException` which you can then catch and do something different.

https://github.com/statamic/cms/blob/2c6fedaeff043877e709c153cf6668d271980b92/src/Http/Controllers/GlideController.php#L123
https://github.com/statamic/cms/blob/2c6fedaeff043877e709c153cf6668d271980b92/src/Http/Controllers/GlideController.php#L102-L104
